plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3699,6 +3703,36 @@ public void testModifyingOldSnapshotIsNotPossible()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCtasFromAVersionedTable()
```

Comment:
nit: Avoid abbreviation
```suggestion
    public void testCreateTableAsSelectFromVersionedTable()
```